test-utils/src/dev/db.rs

```diff
@@ -572,7 +572,14 @@ impl Drop for CockroachInstance {
             }
             #[allow(unused_must_use)]
             if let Some(temp_dir) = self.temp_dir.take() {
-                temp_dir.close();
+                /*
```

Comment:
This is mentioned in the PR comments: we now deliberately save the database directory if you don't explicitly clean it up after a successful test.